FormidableLabs/nextjs-sanity-fe
# Remove graphql

First round of removing GraphQL - this PR migrates any GraphQL queries to groq, removes references of GraphQL in documentation, removes GQL packages, updates e2e tests with current data structure, and adds types for groq queries.

TO TEST:
run the app `pnpm local`
you can manually ensure each page is loading correctly and all the data is rendered
Run `real` e2e tests `pnpm run test:e2e-start`

fixes: #162
